src/mmap.rs

```diff
@@ -377,13 +377,19 @@ pub struct GuestMemoryMmap {
     regions: Vec<Arc<GuestRegionMmap>>,
 }
 
-impl GuestMemoryMmap {
-    /// Creates an empty `GuestMemoryMmap` instance.
-    pub fn new() -> Self {
+impl Default for GuestMemoryMmap {
```

Comment:
How about #[derive(Default)]? I tried "#[derive(Default)]" today and seems it works.

Reply:
Better. Done.